src/isolate_create_params.rs

```diff
@@ -199,7 +207,7 @@ pub(crate) mod raw {
   impl Default for CreateParams {
     fn default() -> Self {
       let size = unsafe { v8__Isolate__CreateParams__SIZEOF() };
-      assert_eq!(size_of::<Self>(), size);
+      assert!(size <= size_of::<Self>());
```

Comment:
It's a bit unfortunate that we will no longer find out about `CreateParams` changes at compile time. 
We always use LLVM's libc++, so is it really not possible to keep the struct sizes on both sizes exactly in sync?

Reply:
If it turns out that maintaining a Rust definition of `CreateParams` is too burdensome because it changes a lot, we should move away from it and add bindings for C++ `new/delete CreateParams()` and add getters/setters for the various properties.

Reply:
> We always use LLVM's libc++

`env GN_ARGS="use_custom_libcxx=false" V8_FROM_SOURCE=1 cargo build` works, AFAICT, and I expect someone somewhere sooner or later will want to use that.

Another problem is that the size of vectors is different in libstdc++ debug builds.

Reply:
> `env GN_ARGS="use_custom_libcxx=true" V8_FROM_SOURCE=1 cargo build` works

Not really a surprise since "use_custom_libcxx" is the default. The "custom libcxx" refers to LLVM libc++.

Using `use_custom_libcxx=false` most certainly does *not* work on Windows, I can't speak to what the situation is on Linux but I suspect you'd have to do some work there too.

Reply:
Haha, I even edited my comment to change `false` to `true` because I wasn't sure which one did what.

Both values produce a working build (seemingly, anyway, on linux), so yeah, I don't think it's safe to assume too much about STL type sizes.